source/assets/stylesheets/_variables.scss

```diff
@@ -11,6 +11,7 @@ $orange: #FDAD2B;
 $yellow: #FED336;
 
 $black: #000000;
+$very-dark-gray: #333333;
```

Comment:
Hah, yeah I wasn't very happy about that name neither; changed to lighter black here: ec28286